docs/community/com-troubleshooting/com-issue-memory-processes.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Performance issues
```

Comment:
Do we need this page? It's essentially the same info that's already broken out for mac and linux. I also feel like "Performance issues" is somewhat misleading as it's just one issue. Performance may also confuse readers here as it generally deals with the speed at which things are done. however the issue here stops featurebase, it doesn't impact its performance. Just my 2 cents

Reply:
I can frag it no problem :)

Reply:
Page fragged but meant I needed to update that weird link in old-size-featurebase-database.md which was a link under the ingester.

FYI - I'm going to hit that page next so the whole thing is going to be rewritten.